### PR TITLE
Improvement/holo 725 add test to make sure token uri is the same on both chains after bridging 

### PR DESCRIPTION
**Pull request by BravoNatalie**
## Describe Changes

- Change `test/06_cross-chain_minting_tests_l1_l2.ts`

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] Code styles have been enforced
- [x] All Hardhat tests are passing

### CODE REVIEW

File: test/06_cross-chain_minting_tests_l1_l2.ts [hunk 1]
```diff
@@ -1,64 +1,24 @@
...
-  adminCall,
 } from '../scripts/utils/helpers';
-import {
-  HolographERC20Event,
-  HolographERC721Event,
-  HolographERC1155Event,
-  ConfigureEvents,
-} from '../scripts/utils/events';
-import ChainId from '../scripts/utils/chain';
-import {
```
Comment (alexanderattar) [inline test/06_cross-chain_minting_tests_l1_l2.ts:37]: Thanks for removing unused imports

File: test/06_cross-chain_minting_tests_l1_l2.ts [hunk 2]
```diff
@@ -399,7 +359,7 @@ describe('Testing cross-chain minting (L1 & L2)', async function () {
         // temporarily set MockLZEndpoint as messaging module, to allow for easy sending
         await l2.operator.setMessagingModule(l2.mockLZEndpoint.address);
         // make call with mockLZEndpoint AS messaging module
-        await l2.mockLZEndpoint.crossChainMessage(l2.operator.address, getLzMsgGas(payload), payload, {
+        await l2.mockLZEndpoint.crossChainMessage(l2.operator.address, getLzMsgGas(String(payload)), payload, {
```
Comment (alexanderattar) [inline test/06_cross-chain_minting_tests_l1_l2.ts:362]: for these existing tests, why was the conversion necessary?

File: test/06_cross-chain_minting_tests_l1_l2.ts [hunk 3]
```diff
@@ -1135,6 +1095,97 @@ describe('Testing cross-chain minting (L1 & L2)', async function () {
...
+            .executeJob(payload, { gasLimit: gasEstimates.estimatedGas });
+
+          const tokenURIAfter = await l2.sampleErc721Enforcer
+            .attach(l1.sampleErc721Holographer.address)
+            .tokenURI(secondNFTl1);
+
+          expect(tokenURIBefore).to.be.equal(tokenURIAfter);
+        });
+
+        //TODO
```
Comment (alexanderattar) [inline test/06_cross-chain_minting_tests_l1_l2.ts]: is this still a TODO?

**Review by sogoiii — APPROVED** [review]
👍

File: test/06_cross-chain_minting_tests_l1_l2.ts (hunk 2 above)
Comment (BravoNatalie) [reply test/06_cross-chain_minting_tests_l1_l2.ts:362]: it was not necessary, but the linter was reporting errors.

File: test/06_cross-chain_minting_tests_l1_l2.ts (hunk 3 above)
Comment (BravoNatalie) [reply test/06_cross-chain_minting_tests_l1_l2.ts]: oww, I forgot to remove it

**Pull request merged**